conda/Dockerfile

```
@@ -1,6 +1,6 @@
-FROM continuumio/miniconda3:4.7.12
+FROM continuumio/miniconda3
 
-ARG ROOT_VERSION=6.20.00
+ARG ROOT_VERSION=6.22.00
 ARG PYTHON_VERSION=3.7
```

Time to use Python 3.8?
```suggestion
ARG PYTHON_VERSION=3.8
```

---

of course, thanks!